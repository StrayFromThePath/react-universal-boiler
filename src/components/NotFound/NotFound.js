// @flow
import React from 'react';
import type { Node } from 'react';

const NotFound = (): Node => {
  return (
    <div>
      <h2 className="header">Not Found</h2>
    </div>
  );
};

export default NotFound;
