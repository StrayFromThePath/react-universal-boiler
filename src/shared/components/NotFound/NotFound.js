import React from 'react';

const NotFound = props => {
  return (
    <div>
      <h2 className="header">Not Found</h2>
    </div>
  );
};

NotFound.displayName = 'NotFound';

export default NotFound;
