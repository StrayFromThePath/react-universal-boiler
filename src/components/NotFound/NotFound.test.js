import React from 'react';
import renderer from 'react-test-renderer';
import NotFound from './NotFound';

jest.useFakeTimers();
Date.now = jest.fn(() => 1482363367071);

it('renders correctly', () => {
  const tree = renderer.create(<NotFound />).toJSON();
  expect(tree).toMatchSnapshot();
});
