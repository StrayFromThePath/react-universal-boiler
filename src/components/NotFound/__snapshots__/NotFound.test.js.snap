// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <h2
    className="header"
  >
    Not Found
  </h2>
</div>
`;
